miden/tests/integration/stdlib/crypto/fri.rs

```diff
@@ -77,3 +77,40 @@ fn test_decorator_ext2intt(in_poly_len: usize, blowup: usize) {
     let test = build_test!(source, &iu64s);
     test.expect_stack(&ou64s);
 }
+
+#[test]
+fn test_verify_remainder_64() {
+    let source = "
+    use.std::crypto::fri
+
+    proc.helper.32
+        locaddr.31
+        repeat.32
+            movdn.4
+            dup.4
+            mem_storew
+            dropw
+            sub.1
+        end
+        drop
+
+        locaddr.0
+        exec.fri::verify_remainder_64
+    end
+
+    begin
+        exec.helper
+    end
+    ";
+
+    let poly = rand_utils::rand_vector::<Ext2Element>(8);
+    let twiddles = fft::get_twiddles(poly.len());
+    let evals = fft::evaluate_poly_with_offset(&poly, &twiddles, Felt::ONE, 8);
+
+    let ifelts = Ext2Element::as_base_elements(&evals);
+    let iu64s = ifelts.iter().map(|v| v.as_int()).collect::<Vec<u64>>();
+
+    let test = build_test!(source, &iu64s);
+    let res = test.execute();
+    assert!(res.is_ok());
```

Comment:
Ideally, we should also test the failure case (i.e., when advice provider returns incorrect data). I'm not sure how to do this yet - so, maybe we should create an issue for this.

Reply:
Absolutely, I've raised an issue https://github.com/0xPolygonMiden/miden-vm/issues/660